src/dmd/e2ir.d

```diff
@@ -2024,7 +2024,11 @@ extern (C++) class ToElemVisitor : Visitor
             elem *es2 = toElem(ie.e2, irs);
             es2 = addressElem(es2, ie.e2.type);
             e = el_param(es1, es2);
-            elem *ecount = el_long(TYsize_t, t1.size());
+            elem *ecount;
+            if (t1.ty == TY.Tfloat80)
```

Comment:
This could be a ternary + comment and link to issue appreciated since the glue code is horrible as it is..